docs-gh-pages/consensus.md

```diff
@@ -0,0 +1,91 @@
+- **SOE** - signed observation edge (see: )
+- **data proposal** - transactions (and other checkpoint block data) proposed by the facilitator to include in the new consensus round
+
+### Consensus round
+1. Start consensus
+2. Broadcast and gather consensus data proposals
+3. Merge, broadcast and gather consensus block proposals
+4. Select a majority block proposal and merge facilitators' signatures
+5. Accept a finished block and broadcast it to non-facilitators
+
+### Stages
+
+#### 1. Start consensus
+`ConsensusScheduler` runs `consensusManager.startOwnConsensus()` every **n** seconds
+(*default value: 10s*, configurable under `constellation.consensus.start-own-interval`).
+To start a new round, node must be in one of valid node states. Otherwise, creating new round will be skipped.
+Valid node states for starting new consensus round are `Ready` and `SnapshotCreation`.
+
+When new round can be created, owner creates one by using pending transactions (up to `maxTransactionThreshold`), tip SOE
+and selects final facilitators. Peers which can take part in the consensus round must have `nextSnapshotHeight` below
+the proposed tip. Otherwise, the selected node will not be able to accept the final checkpoint block. Facilitators are being
+selected randomly taking into account above preconditions.
+
+**Dummy transaction** is a transaction pulled to the consensus round if node has no real transactions. Node should always
+be able to create a block, even if the block is empty. To distinguish empty blocks, there must be at least one dummy transaction
+which is equivalent to the real transaction but carries 0 amount.
+
+#### 2. Broadcast and gather consensus data proposals
+Owner sends the `NotifyFacilitators` message to all selected facilitators and waits for the response. If facilitator is able
+to start a consensus, responds with success message and creates a corresponding round. Facilitator cannot start a consensus
+if proposed tip is below the next snapshot height or is in the incorrect state (i.e. leaving from the cluster).
+When all facilitators have been notified and started rounds successfully, each one pulls transactions, similarly as owner did,
+and broadcasts it to all consensus round's participants. In parallel, each participant gathers proposals from others and keeps it locally.
+
+#### 3. Merge, broadcast and gather consensus block proposals
+When node received all proposals, it merges them together to create a block proposal and then broadcasts it to others.
+Such a block proposal is being signed by the proposer only. The same process happens on every node taking part in a consensus round,
+hence when data used for block proposal creation are the same, block proposals will be equal (except signatures which obviously must be different). 
+
+#### 4. Select a majority block proposal and merge facilitator's signatures
+When node received all block proposals, it verifies the correctness of proposals. The majority block is being chosen
```

Comment:
I think you can change "received" to "receives" in couple of places to be more consistent with the style of the whole doc.

Reply:
but if verifies it when received all proposals

Reply:
and `receives` is used once only in the correct form